lib/flutter_state_render_dialog/lib/flutter_state_render_dialog.dart

```diff
@@ -1,24 +1,26 @@
 import 'package:flutter/material.dart';
 import 'package:lottie/lottie.dart';
 
-dialogRender({
-  required BuildContext context,
-  required StateRenderType stateRenderType,
-  required String message,
-  required String title,
-  retryAction,
-  Widget? child,
-}) {
-  showDialog(
-    context: context,
-    builder: (context) => StateRender(
-      stateRenderType: stateRenderType,
-      message: message,
-      title: title,
-      retryAction: retryAction,
-      child: child,
-    ),
-  );
+class DialogRender {
```

Comment:
It is preferable to use function over class in this case, as it may cause some problems with some users in old versions